testProjectAndroid/build.gradle

```diff
@@ -105,70 +123,70 @@ afterEvaluate {
   }
 
   test.doLast {
-    org.junit.Assert.assertEquals(
-        ['generateArmFreeappDebugAndroidTestProto', 'generateArmFreeappDebugProto',
+    assert ['generateArmFreeappDebugAndroidTestProto', 'generateArmFreeappDebugProto',
          'generateArmFreeappReleaseProto', 'generateArmRetailappDebugAndroidTestProto',
          'generateArmRetailappDebugProto', 'generateArmRetailappReleaseProto',
          'generateX86FreeappDebugAndroidTestProto', 'generateX86FreeappDebugProto',
          'generateX86FreeappReleaseProto', 'generateX86RetailappDebugAndroidTestProto',
-         'generateX86RetailappDebugProto', 'generateX86RetailappReleaseProto'] as Set,
-         protobuf.generateProtoTasks.all().collect({ it.name }) as Set)
-    org.junit.Assert.assertEquals(
-        ['generateArmFreeappDebugAndroidTestProto',
+         'generateX86RetailappDebugProto', 'generateX86RetailappReleaseProto'] as Set ==
+         protobuf.generateProtoTasks.all().collect({ it.name }) as Set
+
+    assert ['generateArmFreeappDebugAndroidTestProto',
```

Comment:
It's not common to use `assert` in tests. I would stay away from it, as it's not obvious whether it's enabled or not, although it's enabled in this case as I tested. Please revert to the JUnit assertions.